build/Dockerfile

```diff
@@ -137,15 +138,16 @@ apt-get install -y --no-install-recommends \
     xxd
 EOF
 
+# Install Rust to compile foundry from source because we use a specific tag.
+ARG RUST_VERSION
+RUN curl https://sh.rustup.rs -sSf | bash -s -- --default-toolchain ${RUST_VERSION} -y
+ENV PATH="/root/.cargo/bin:${PATH}"
+
 # Install Foundry.
 RUN curl -L https://foundry.paradigm.xyz | bash
-ENV PATH="~/.foundry/bin:${PATH}"
```

Comment:
This didn't work; that is why we needed to create the symbolic link. With the fix, we don't need the link anymore.